content/en/docs/talos/configuration/talm.md

```
@@ -19,6 +19,26 @@ cd cluster1
 talm init --preset cozystack
 ```
 
+{{% alert color="warning" %}}
+If you use keycloak.
+
+From v0.6.6 Talm:
+- in `cluster1/templates/_helpers.tpl` replace  `keycloak.example.com` to your domain.
+
+Before v0.6.6 Talm:
+- Add in template args manualy:
+```
```

Markdownlint (0.35.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)